src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,139 @@
+import { html } from "lit-html";
+import "../button";
+import "../alert";
+import styles from "./styles.scss";
+import { stylesheet } from "../../../.storybook/utils.js";
+
+export default {
+  title: "custom-elements/vl-alert",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  args: {
+    title: "Opgelet!",
+    icon: "warning",
+    size: "normal",
+    type: "info",
+    closable: false,
+  },
+  argTypes: {
+    title: {
+      name: "data-vl-title",
+      type: { summary: "string" },
+      description: "Attribuut wordt gebruikt om de titel te bepalen.",
+      table: {
+        defaultValue: { summary: "" },
+      },
+    },
+    icon: {
+      name: "data-vl-icon",
+      type: "select",
+      options: ["warning", "check", "info-circle"],
+      description: "Attribuut wordt gebruikt om het icoon type te bepalen.",
+      table: {
+        type: { summary: "string" },
+        defaultValue: { summary: "" },
+      },
+    },
+    size: {
+      name: "data-vl-size",
+      type: "select",
+      options: ["normal", "small"],
+      description:
+        "Attribuut activeert een variant van de waarschuwing maar kleiner.",
+      table: {
+        type: { summary: "string" },
+        defaultValue: { summary: "" },
+      },
+    },
+    type: {
+      name: "data-vl-type",
+      type: "select",
+      options: ["info", "success", "warning", "error"],
+      description:
+        "Attribuut bepaalt de soort van waarschuwing, foutmelding, probleemmelding of succesmelding.",
+      table: {
+        type: { summary: "string" },
+        defaultValue: { summary: "" },
+      },
+    },
+    closable: {
+      name: "data-vl-closable",
+      type: { summary: "boolean" },
+      description:
+        "Attribuut wordt gebruikt om de optie toe te voegen om de waarschuwing te sluiten door op het sluit icoon te klikken in de rechterbovenhoek.",
+      table: {
+        defaultValue: { summary: "false" },
+      },
+    },
+  },
+};
+
+export const Default = ({
+  closable,
+  disabled,
+  icon,
+  title,
+  size,
+  type,
+}) => html`
+  <vl-alert
+    ?data-vl-closable=${closable}
+    ?data-vl-disabled=${disabled}
+    data-vl-icon=${icon}
+    data-vl-title=${title}
+    data-vl-size=${size}
+    data-vl-type=${type}
+  >
+    <p>
+      U heeft geen rechten om deze actie uit te voeren.
+      <a href="#">Vraag rechten aan</a>.
+    </p>
+  </vl-alert>
+`;
+
+export const AlertWithButton = ({
+  closable,
+  disabled,
+  icon,
+  title,
+  size,
+  type,
+}) => html`
+  <vl-alert
+    ?data-vl-closable=${closable}
+    ?data-vl-disabled=${disabled}
+    data-vl-icon=${icon}
+    data-vl-title=${title}
+    data-vl-size=${size}
+    data-vl-type=${type}
+  >
+    <p>
+      U heeft geen rechten om deze actie uit te voeren.
```

Comment:
Ik zou voor de stories het a element weglaten, en in de p een control 'content (for demo purposes)' maken, zodat je in de story zelf snel een persoonlijke alert kan samenstellen. In het geval van deze story zou ik ook button text (for demo purposes) toevoegen.

Reply:
Done.